string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,18 @@
+/* exported isVowel */
+function isVowel(char) {
+  if (char.length === 1) {
+    var vowels = 'aeiouAEIOU';
+    var isVowel = vowels.indexOf(char) >= 0;
```

Comment:
Try doing this without using `indexOf` 🤓

Reply:
done 👍